src/main/java/uk/gov/ch/model/update/CorporateTrusteeData.java

```diff
@@ -0,0 +1,211 @@
+package uk.gov.ch.model.update;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+@Entity
+public class CorporateTrusteeData {
+
+    @Column(name = "TRUSTEE_ID")
+    @JsonProperty("trusteeId")
+    private Long trusteeId;
+
+    @Column(name = "TRUSTEE_NAME")
+    @JsonProperty("trusteeName")
+    private String trusteeName;
+
+    @Column(name = "REGISTER_LOCATION")
+    @JsonProperty("registerLocation")
+    private String registerLocation;
+
+    @Column(name = "REGISTRATION_NUMBER")
+    @JsonProperty("registrationNumber")
+    private String registrationNumber;
+
+    @Column(name = "LAW_GOVERNED")
+    @JsonProperty("lawGoverned")
+    private String lawGoverned;
+
+    @Column(name = "LEGAL_FORM")
+    @JsonProperty("legalForm")
+    private String legalForm;
+
+    @Column(name = "COUNTRY")
+    @JsonProperty("country")
+    private String country;
+
+    @Column(name = "MODIFICATION_ID")
```

Comment:
Theres a few fields here not needed, looking at the query there should be 11 fields returned but in this file there are 14